src/features/patients/pages/PatientForm.tsx

```diff
@@ -98,6 +131,14 @@ export const PatientForm: FC = () => {
     }
   };
 
+  // The on submit function of the form which handles events such as focusing
+  // fields which are not filled is triggered by clicking a button which has a
+  // type of 'submit' as a child of the form. Since we have two form components
+  // and we want each one to be validated with a single button, there is a
+  // button child for each form. Where the second form is triggering a simulated
+  // click on the first forms button, to trigger the on submit function for that form.
+  const buttonRef = useRef<HTMLButtonElement | null>(null);
```

Comment:
OOOk .. so hopefully the comment makes sense.

Also played around with some imperative usage of the form- it's confusing with the imperative usage of the form component mixed with events. The event seems to give the 'retarget' of required fields which aren't valid which is a bit of a bonus which I think is just an html thing, but it doesn't effect 'invalid state' form inputs which we would have to come up with our own mechanism to do